lib/menu.rb

```diff
@@ -0,0 +1,26 @@
+class Menu
+  attr_reader :menu_list
+  def initialize
+    @menu_list = {
+      tortilla: 4,
+      gazpacho: 3,
+      pasta: 5,
+      pizza: 5,
+      lasagna: 6,
+      salad: 4,
+      coke: 2
+    }
+  end
+
+  def display_menu
+    menu_list.each_pair { |dish, price| p "Dish: #{dish}, price: £#{price}" }
```

Comment:
Having display logic like this in the same class as other, heavier business logic is generally a good sign that you need to create another object - maybe think about a MenuPrinter?